docs/maven-plugin.md

```diff
@@ -0,0 +1,174 @@
+# Usage
+
+Add the following profiles to your pom.xml:
+
+```
+  <profile>
+    <id>check</id>
+    <build>
+      <pluginManagement>
+        <plugins>
+          <plugin>
+            <groupId>org.openhab.tools</groupId>
+            <artifactId>static-code-analysis</artifactId>
+            <version>${sat.version}</version>
+            <executions>
+              <execution>
+                <phase>verify</phase>
+                <goals>
+                  <goal>checkstyle</goal>
+                  <goal>pmd</goal>
+                  <goal>findbugs</goal>
+                  <goal>report</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </pluginManagement>
+    </build>
+  </profile>
+  <profile>
+    <id>check-bundles</id>
+    <activation>
+      <file>
+        <exists>META-INF/MANIFEST.MF</exists>
+      </file>
+    </activation>
+    <build>
+      <plugins>
+        <plugin>
+            <groupId>org.openhab.tools</groupId>
+            <artifactId>static-code-analysis</artifactId>
+        </plugin>
+      </plugins>
+    </build>
+  </profile>
+```
+
+ Execute `mvn clean install -P check` from the root of your project.
+
+ Reports are generated for each module individually and can be found in the `target/code-analysis` directory. The merged report can be found in the root target directory.

```

Comment:
As a user, I will really appreciate a screenshot, sample HTML or the last report of openhab build (one of the three) in order to see how the report looks like and does it worth to invest some time to run it.